dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java

```diff
@@ -67,7 +67,7 @@ public static FeatureReplacementAction ft11metrics20Features(String server) {
      * @return the RepeatTests rule
      */
     public static RepeatTests repeatDefault(String server) {
-        return repeat(server, TestMode.FULL, MicroProfileActions.MP40, MicroProfileActions.MP20);
+        return repeat(server, TestMode.FULL, MicroProfileActions.MP50, MicroProfileActions.MP40, MicroProfileActions.MP20);
     }
```

Comment:
Please update the javadoc for the methods you've changed in this file.

Reply:
done